ODataConnectedService/src/Templates/ODataT4CodeGenerator.cs

```diff
@@ -237,7 +237,7 @@ public string MetadataDocumentUri
         {
             value = uri.Scheme + "://" + uri.Authority + uri.AbsolutePath;
             value = value.TrimEnd('/');
-            if (!value.EndsWith("$metadata", StringComparison.CurrentCulture))
+            if (!value.EndsWith("$metadata"))
```

Comment:
Maybe this change was not meant for checkin?

Reply:
This change was from a previous commit on the tt file but the generated cs file wasn't committed

Reply:
If it ain't for this particular PR (Replace Dictionary with IDictionary in ByKey method) then it should probably be in a different PR...